001-core.php

```diff
@@ -39,3 +39,17 @@ function wpcom_vip_disable_core_update_cap( $caps, $cap ) {
 	}
 	return $caps;
 }
+
+/**
+ * Diable tests in the Site Health (AKA site status) tool page
+ *
+ * By default, WordPress runs a series of tests on the Site Health tool
+	* page in wp-admin. Some tests, such as the Background Updates test, are
+	* irrelevant because the updates are managed by the VIP team.
+	* This disables all unnecessary tests.
+ */
+function remove_background_updates_test( $tests ) {
+	unset( $tests['async']['background_updates'] );
```

Comment:
A few suggested changes here:

- Fix the indents in the docblock
- Prefix the function name and make it more generic (since we may remove other tests too?)
- Add `isset` checks to be a bit defensive.
- Move the comment about Background Updates inline to make it more contextual (this way, if we change the the behavior, we don't need to remember to update the docblock too).

```suggestion
 * By default, WordPress runs a series of tests on the Site Health tool
 * page in wp-admin. This disables all irrelevant or unnecessary tests.
 */
function vip_disable_unnecessary_site_health_tests( $tests ) {
	// Disable "Background Updates" test.
	// WordPress updates are managed by the VIP team.
	if ( isset( $tests['async'] ) && isset( $tests['async']['background_updates'] ) ) {
		unset( $tests['async']['background_updates'] );
	}
```

Reply:
Ah got it! I thought we were having each test removal be its own function (for separation purposes) but I guess that could get quite messy!

Fixed + added all the points 👍